### PR TITLE
Adds a view-restricting overlay to the welding mask

### PR DESCRIPTION
**Pull request by Shroopy**
## About the PR 
Adds a restrictive overlay that keeps users from seeing more than one and a half tiles away with a welding mask flipped down.
## Why's this needed? 
Adds more of a reason to keep the welding mask flipped up. Also, asked for by ZeWaka.
## Changelog

```
(u)Shroopy:
(*)Welding mask now has a view-restricting overlay when flipped down.
```

### CODE REVIEW

File: code/obj/item/clothing/helmets.dm [hunk 1]
```diff
@@ -508,6 +508,9 @@
 		setProperty("meleeprot_head", 4)
 		setProperty("disorient_resist_eye", 0)
 
+	unequipped(var/mob/living/user)
```
Comment (ZeWaka) [inline code/obj/item/clothing/helmets.dm:511]: you'll also probably want to do `equipped()`

**Comment by Shroopy** [comment]
Sorry for all the weird commits, but the git graph should be nice and pretty now. Also, it no longer checks every tick on sight.dm, now it only sets the overlay when needed.

**Comment by ZeWaka** [comment]
why did you make it stronger?

**Comment by Shroopy** [comment]
I didn't, it's a multiplier, so 0.5 is half-bright and .75 is three-quarters bright.

**Comment by ZeWaka** [comment]
> 
> 
> I didn't, it's a multiplier, so 0.5 is half-bright and .75 is three-quarters bright.

ah, I was mislead by your commit message :flooshed:

**Comment by mybluecorners** [comment]
voicing my not-support of an obscuring screen overlay for the welding mask. 
it's already got some 'balance' flipped up vs. down because when flipped up it provides more head armor and flipped down it provides eye protection.

**Comment by Shroopy** [comment]
Bump? I made the darken less strong, which should fit the requested change.

**Pull request closed without merging**